### PR TITLE
Alison/commissions

### PR DESCRIPTION
**Pull request by adowski**
## Feature Name
`Commissions` table

### Related PRs
None

### Migrations
Created relationships between `Commissions`, `Artists`, and `Buyers`
Added `price` and `comment` attributes
Added `artist_id` and `buyer_id` attributes

### Tests Performed, Edge Cases
None

### Screenshots
None

CC: @clairelin135

### CODE REVIEW

File: test/fixtures/commissions.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  price: 1
+  comment: MyText
+
+two:
```
Comment (JasonBaoZ) [inline test/fixtures/commissions.yml]: What is this for?
